flowetl/tests/integration/conftest.py

```diff
@@ -18,34 +19,41 @@
 from pendulum import now, Interval
 from airflow.models import DagRun
 from sqlalchemy import create_engine
-from sqlalchemy.orm import sessionmaker, scoped_session
-from docker import from_env, APIClient
+from sqlalchemy.orm import sessionmaker
 from docker.types import Mount
 from shutil import rmtree
 
+here = os.path.dirname(os.path.abspath(__file__))
+logger = logging.getLogger("flowetl-tests")
+
+
+@pytest.fixture(scope="session")
+def flowetl_mounts_dir():
+    return os.path.abspath(os.path.join(here, "..", "..", "mounts"))
```

Comment:
Just a note for the future that it would be preferable if this stuff used tmpdirs instead of relying on empty directories we've tricked git into keeping around.